controllers/auth/socials/facebook.js

```diff
@@ -0,0 +1,26 @@
+import dotenv from "dotenv";
+import passport from "passport";
+import facebookStrategy from "passport-facebook";
+import createUserFromSocial from "./createUserFromSocial";
+
+dotenv.config();
+
+const FacebookStrategy = facebookStrategy.Strategy;
+passport.use(
+  new FacebookStrategy(
+    {
+      clientID: process.env.FACEBOOK_CLIENT_ID,
+      clientSecret: process.env.FACEBOOK_CLIENT_SECRET,
+      callbackURL: "http://localhost:3000/api/v1/auth/facebook/callback",
```

Comment:
make the URL base an environment variable so that this will work both in development as well as in production.